resources/js/components/EideasyWidget.vue

```diff
@@ -0,0 +1,47 @@
+<script>
+import '@eid-easy/eideasy-widget';
+
+export default {
+    props: {
+        docId: String,
+        baseUrl: String,
+        clientId: String,
+    },
+    data() {
+        return {
+            apiEndpoints: {
+                identityStart: () => this.baseUrl + '/api/identity/start',
+                identityFinish: () => this.baseUrl + '/api/identity/finish',
+            },
```

Comment:
We can remove the `identityStart`  and `identityFinish` as it's not needed for signing flows.

Reply:
Let's add the following to the `apiEndpoints` config:
`base: () => this.baseUrl,` This way it'll use the correct base url for making api requests.

Reply:
I've simplified it even further, and basically `apiEndpoints` configuration is not even necessary here, since if you are using `sandbox` mode, API requests would go to `test.` environment anyways. 

I could not find any implementation for `apiEndpoints.base` support in [Vue.js](https://github.com/eideasy/eideasy-widget/blob/master/src/App.vue#L78) widget component, contrary to [Signature.js](https://github.com/eideasy/signature-js/blob/master/src/Signature.ts#L7), so it should work out of the box. 

Fixup: [24c87cf0253bd059b5930b50fae84657baaf6692]